ui/app/routes/vault/cluster/logout.js

```diff
@@ -16,14 +16,15 @@ export default Route.extend(ModelBoundaryRoute, {
   }),
 
   beforeModel() {
-    let authType = this.auth.getAuthType();
+    const authType = this.auth.getAuthType();
+    const baseUrl = window.location.origin;
     this.auth.deleteCurrentToken();
     this.controlGroup.deleteTokens();
     this.namespaceService.reset();
     this.console.set('isOpen', false);
     this.console.clearLog(true);
     this.flashMessages.clearMessages();
     this.permissions.reset();
-    this.replaceWith('vault.cluster.auth', { queryParams: { with: authType } });
```

Comment:
Just an fyi that you can use the router service to get a full url by name (but I don't see the auth route changing anytime soon 😂): 
https://api.emberjs.com/ember/3.27/classes/RouterService/methods/urlFor?anchor=urlFor

Reply:
Thanks for the tip! You never know 😅

Reply:
For some reason when I import the router service and try to get the path from it, it breaks some test page selectors 🤔 Spent a whole day trying to figure out what was going on with no luck, so going with the hardcoded auth link for now 😞  @meirish if you have any ideas of what might be going on let me know, but this should work for now